client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -37,6 +37,7 @@ export const siteSetupFlow: Flow = {
 		const isEnabledFTM = isEnabled( 'signup/ftm-flow-non-en' ) || isEnglishLocale;
 
 		return [
+			'launchpad',
```

Comment:
**Note:**

This is temporary. I only placed launchpad here in the site-setup-flow to make it accessible in the UI. In reality, it would live in the link-in-bio, podcast, and newsletter flows.